build.sbt

```diff
@@ -135,6 +135,11 @@ lazy val free = project.dependsOn(macros, core)
   .settings(moduleName := "cats-free")
   .settings(catsSettings)
 
+lazy val state = project.dependsOn(macros, core, free, tests % "test -> test")
+  .settings(moduleName := "cats-state")
+  .settings(catsSettings)
+  .settings(noPublishSettings)
```

Comment:
Oh, good catch. I would be fine publishing it.

Want to submit a small PR fixing this?